project/src/js/ui/round/roundCtrl.js

```diff
@@ -404,7 +404,11 @@ export default function controller(cfg, onFeatured, onTVChannelChange, userTv, o
     if (this.chat) this.chat.onReload(rCfg.chat);
     if (this.data.tv) rCfg.tv = this.data.tv;
     if (this.data.userTV) rCfg.userTV = this.data.userTV;
+    let tournament = this.data.tournament;
+
     this.data = data(rCfg);
+
+    this.data.tournament = tournament;
```

Comment:
Much better. Made this change.

On Mon, May 9, 2016 at 4:39 AM, Vincent Velociter <notifications@github.com>
wrote:

> In project/src/js/ui/round/roundCtrl.js
> <https://github.com/veloce/lichobile/pull/327#discussion_r62465402>:
>
> >      this.data = data(rCfg);
> > +
> > +    this.data.tournament = tournament;
>
> Actually it's better to be consistent with the lines above and write:
>
> if (this.data.tournament) rCfg.tournament = this.data.tournament;
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/veloce/lichobile/pull/327/files/a758f92fb272d071df58e85539eb0b4a1ebaaa70#r62465402>
>